.github/workflows/workflow.yml

```diff
@@ -390,9 +393,10 @@ jobs:
         # Restore kubeconfig from remote docker host.
         mkdir -p $HOME/.kube
         scp linkerd-docker:/tmp/kind-config-$KIND_CLUSTER $HOME/.kube
-        export KUBECONFIG=$(bin/kind get kubeconfig-path --name=$KIND_CLUSTER)
+        export KUBECONFIG=$HOME/.kube/kind-config-$KIND_CLUSTER
```

Comment:
similar to previous comment:
```bash
export KUBECONFIG=$HOME/.kube/kind-config-$KIND_CLUSTER
scp linkerd-docker:/tmp/kind-config-$KIND_CLUSTER $KUBECONFIG
```

...or, going a bit further, it looks like there is a new subcommand to re-export a kubeconfig file without creating a cluster. it's possible you could remove the `scp` commands to copy the kubeconfig file back and forth, and instead do:
```bash
export KUBECONFIG=$HOME/.kube/kind-config-$KIND_CLUSTER
kind export kubeconfig --name=$KIND_CLUSTER
```

this is a bit beyond the scope of a kind version bump, so ok to explore in a subsequent PR.